client/components/content-containers/tcc-exam/edit/AssessmentGroup.vue

```diff
@@ -15,128 +15,120 @@
           step="15">
       </div>
     </div>
-    <h3>Question group {{ toLetter(position) }}</h3>
+    <h3>Question group {{ position | toLetter }}</h3>
     <h4>Introduction</h4>
-    <group-introduction :group="group" />
+    <group-introduction
+      @saveElement="$emit('saveElement', $event)"
+      @reorderElement="$emit('reorderElement', $event)"
+      @deleteElement="$emit('deleteElement', $event)"
+      :group="group"
+      :tes="tes" />
```

Comment:
Good idea but I think it should be a separate PR since the changes would affect code outside the scope of this PR. We should also see if we want to make changes to the data layer.